.NET/Microsoft.Recognizers.Text.DataTypes.TimexExpression/TimexRegex.cs

```diff
@@ -1,49 +1,54 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
```

Comment:
Please don't strip copyright notices.

Reply:
ah sorry, copy-pasting late at night - this was just a working version from my own project